src/main/java/io/github/fablabsmc/fablabs/api/fluidvolume/v1/volume/api/FixedSizeFluidVolume.java

```diff
@@ -0,0 +1,13 @@
+package io.github.fablabsmc.fablabs.api.fluidvolume.v1.volume.api;
+
+import io.github.fablabsmc.fablabs.api.fluidvolume.v1.math.Fraction;
+
+/**
+ * a fluid volume who's size is limited
+ */
+public interface FixedSizeFluidVolume extends SingleFluidContainer {
```

Comment:
What's the difference between a `FluidVolume` and a `FluidContainer` at the moment? Is `FluidVolume` for actual data and `FluidContainer` for transfer limits? Should `FluidVolume` and `FluidContainer` be on the same class?

Reply:
FluidVolume is something that carries one type of fluid, a FluidContainer can hold multiple

Reply:
Ahh, makes sense! Definitely feels like a good idea to just directly use the volume if there's only one volume in the container.

Reply:
I'm not satisfied with many of the class names, but if you have better names, please suggest